src/RetireNet.Packages.Tool/Services/RetireServiceOptions.cs

```diff
@@ -9,5 +9,9 @@ public class RetireServiceOptions
         public string Path { get; set; }
 
         public bool AlwaysExitWithZero { get; set; }
+        public string ReportPath { get; set; }
+        public String ReportFormat { get; set; }
+        public bool NoRestore { get; set; }
+        public bool NoColor { get; set; }
```

Comment:
Out of scope of the feature (reporting/reporting formats) being added. Open a new issue, and let's discuss it there instead.

Reply:
Fair enough, I figured the footprint was small enough I could just toss it in :)

Reply:
Yeah, totally agree, and I do vary this myself tbh. Sometimes it's not that important to split. In favor of splitting, I find that it's easier to revert one without the other if needed :) Also, for open source projects, I think it's good to have a issue where one can track a discussion or if that feature is wanted/needed first :)